src/main/java/org/blockjam/core/bungee/BungeeServers.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * This file is part of BlockJamCore, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) 2016, BlockJam <https://blockjam.org/>
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package org.blockjam.core.bungee;
+
+/**
+ * An enumeration of the base servers within the BlockJam network.
+ */
+public enum BungeeServers {
```

Comment:
This enum will eventually have to power to do something like `SURVIVAL_GAMES("sg1", "sg2")` and will be able to auto determine the best server to goto. This will be very useful for a `/join minigame` command`